certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2456,7 +2475,8 @@ public class Script : ScriptBase
 
     if (response.Content?.Headers?.ContentType != null)
     {
-      if ("GetDocuments".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+      if (("GetDocuments".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase)) ||
+      ("GetDocumentsWithDocumentId".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase)))
```

Comment:
both options make use document id, so why suffix with `WithDocumentId`, lets call the new option as `GetDocumentsV2`

Reply:
Yes, agree. Will update this.